FiveCalls/FiveCalls/App.swift

```diff
@@ -8,14 +8,28 @@
 
 import SwiftUI
 
-//@main // TODO: once SwiftUI is done move main here
-//struct FiveCallsApp: App {
-//    @UIApplicationDelegateAdaptor(AppDelegate.self) var appDelegate
-//            
-//    var body: some Scene {
-//        WindowGroup {
-//            Dashboard().environmentObject(appDelegate.appState)
-//        }
-//    }
-//}
+@main
+struct FiveCallsApp: App {
+    @UIApplicationDelegateAdaptor(AppDelegate.self) var appDelegate
+            
+    let store = Store(state: AppState(), middlewares: [appMiddleware()])
+    @AppStorage(UserDefaultsKey.hasShownWelcomeScreen.rawValue) var hasShownWelcomeScreen = false
+    @State var welcomePresented = false
```

Comment:
nit: we've been using a more active nomenclature for these types of properties, otherwise it sounded like a duplicate of `hasShownWelcomeScreen`
```suggestion
    @State var showWelcomeScreen = false
```

Reply:
👍